pkg/replication/deployment_replication.go

```diff
@@ -170,7 +172,7 @@ func (dr *DeploymentReplication) run() {
 			switch event.Type {
 			case eventArangoDeploymentReplicationUpdated:
 				if err := dr.handleArangoDeploymentReplicationUpdatedEvent(event); err != nil {
-					dr.reportDeploymentReplicationErr(err, "Failed to handle deployment replication update")
+					// It should not go into failed phase, maybe CR was not written to Kubernetes.
```

Comment:
Maybe we can put at least some log.Warning() here?

Reply:
added